README.md

```diff
@@ -1,6 +1,6 @@
 # mbtest
 
-Opinionated Python wrapper & utils for the [Mountebank](https://www.mbtest.org/) over the wire test double tool.
+Opinionated Python wrapper & utils for the Mountebank over the wire test double tool.
```

Comment:
Might be better to link to https://github.com/bbyars/mountebank rather than not having a link at all...

```suggestion
Opinionated Python wrapper & utils for the [Mountebank](https://github.com/bbyars/mountebank) over the wire test double tool.
```